pkg/google/gke/disk.go

```diff
@@ -124,3 +126,11 @@ func (d Disk) DiskType() string {
 	}
 	return "persistent_volume"
 }
+
+func (d Disk) UseStatus() string {
+	if len(d.users) == 0 {
+		return "idle"
```

Comment:
Suggestion: turn `idle` and `in-use` into const variables that we can reference. This should help keep the tests and the code inline without much toil and refactors are simpler.